packages/ocephes/ocephes.0.0.1/descr

```diff
@@ -0,0 +1,4 @@
+Bindings to special math functions from the Cephes library.
```

Comment:
Shouldn't there be a `depext` for `Cephes` then (it does not seem to be packaged with Debian though)?